content/en/blog/releases/1-11-0/index.md

```diff
@@ -0,0 +1,217 @@
+```
+
+### Cosign 2.0 Support
+
+Kyverno 1.11 also adds support for Cosign 2.0 and all the behavioral changes in Cosign 2.0 are supported in Kyverno. Tlogs and SCTs are now verified by default and can be turned off using `rekor.ignoreTlogs` and `ctlog.IgnoreSCT` attributes in a policy. Rekor public keys and ctlog public keys can now be specified in policies to verify Tlogs and SCT timestamps without setting up TUF for Sigstore or passing a Rekor URL.
+
+Full support for private Sigstore deployments has also been added. Private Sigstore deployments can be used by passing the TUF root and mirror in the Kyverno deployment or Helm chart.
+
+### Image Verification Cache
+
+Image verification requires multiple network calls and can be time consuming. We have also added caching for image verification that will cache successful image verification outcomes. It is a TTL based cache and the size and TTL configuration can be configured in the deployment. 
+
+**Note:** Users upgrading from Kyverno v1.10 to v1.11 who have image verification policies using cosign will have to explicitly disable Tlogs and SCT verification in their policy using the `rekor.ignoreTlogs` and `ctlog.IgnoreSCT` fields if they did not use Rekor while signing the image.
+
+### Cleanup via TTL label
+
+The cleanup ability was introduced in Kyverno 1.9 via the cleanup type of policy. This release introduced another option to clean up resources via a reserved time-to-live (TTL) label `cleanup.kyverno.io/ttl`. By assigning this label to any resource with required permissions granted to Kyverno, the resource will be removed at the designated time. In this release, the cleanup policy no longer relies on CronJobs to perform the job.
+
+For example, creation of this Pod will cause Kyverno to clean it up after two minutes and without the presence of a cleanup policy.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+ labels:
+   cleanup.kyverno.io/ttl: 2m
+ name: foo
+spec:
+ containers:
+ - args:
+   - sleep
+   - 1d
+   image: busybox:1.35
+   name: foo
+```
+
+Because this is a label, there is opportunity to chain other Kyverno functionality around it. For example, it is possible to use a Kyverno mutate rule to assign this label to matching resources. A validate rule could be written prohibiting, for example, users from the `infra-ops` group from assigning the label to resources in certain Namespaces. Or, Kyverno could generate a new resource with this label as part of the resource definition.
+
+### CLI Refactoring and New Test Schema
+
+In Kyverno 1.11, we have made significant improvements to the CLI, enhancing its stability and usability. A new test manifest schema was introduced to the Kyverno `test` command, now you can validate your `test.yaml` and get helpful error reports during execution. Here’s a snippet of a `test.yaml` and an error is displayed as part of the execution result, note that the full schema such as apiVersion and kind is not supported in this release.
```

Comment:
>note that the full schema such as apiVersion and kind is not supported in this release.

I don't understand what this means.

Also, we're saying `test.yaml` (in code) implying the file name for this new test manifest should be `test.yaml`, but that's not the case, right? It's still expected to be `kyverno-test.yaml`, correct?